docs/guides/use_checkpointing.md

```diff
@@ -88,29 +106,35 @@ Now save the checkpoint. You can add annotations like step number, prefix, and s
 
 When saving a checkpoint, Flax will bookkeep the existing checkpoints based on your arguments. For example, by setting `overwrite=False` in [`flax.checkpoints.save_checkpoint`](https://flax.readthedocs.io/en/latest/api_reference/flax.training.html#flax.training.checkpoints.save_checkpoint), Flax will not automatically save your checkpoint if there is already a step that is equal to or newer than the current one presently in the checkpoint directory. By setting `keep=2`, Flax will keep a maximum of 2 checkpoints in the directory. Learn more in the [API reference](https://flax.readthedocs.io/en/latest/api_reference/flax.training.html#module-flax.training.checkpoints).
 
-```{code-cell} ipython3
+You can start to use Orbax to handle the underlying save by creating an `orbax.Checkpointer`, and pass it into the `flax.checkpoints.save_checkpoint` call.
+
+```python
 # Import Flax Checkpoints.
 from flax.training import checkpoints
 
-from jax.experimental.gda_serialization.serialization import GlobalAsyncCheckpointManager
-
 ckpt_dir = 'tmp/flax-checkpointing'
 
 if os.path.exists(ckpt_dir):
     shutil.rmtree(ckpt_dir)  # Remove any existing checkpoints from the last notebook run.
 
+orbax_checkpointer = orbax.Checkpointer(orbax.PyTreeCheckpointHandler())
 checkpoints.save_checkpoint(ckpt_dir=ckpt_dir,
                             target=ckpt,
                             step=0,
                             overwrite=False,
-                            keep=2)
+                            keep=2,
+                            orbax_checkpointer=orbax_checkpointer)
 ```
 
 ## Restore checkpoints
 
-To restore a checkpoint, use [`flax.training.checkpoints.restore_checkpoint`](https://flax.readthedocs.io/en/latest/api_reference/flax.training.html#flax.training.checkpoints.restore_checkpoint) and pass in the checkpoint directory. Flax will automatically select the latest checkpoint in the directory. You can also choose to specify a step number or the path of the checkpoint file. You can always restore a pytree out of your checkpoints by setting `target=None`.
+To restore a checkpoint, use [`flax.training.checkpoints.restore_checkpoint`](https://flax.readthedocs.io/en/latest/api_reference/flax.training.html#flax.training.checkpoints.restore_checkpoint) and pass in the checkpoint directory. Flax will automatically select the latest checkpoint in the directory. You can also choose to specify a step number or the path of the checkpoint file.
+
+With the Orbax migration on progress, `restore_checkpoint` can automatically identify whether a checkpoint is saved in legacy or orbax version, and restore the pytree correctly.
+
+You can always restore a pytree out of your checkpoints by setting `target=None`.
```

Comment:
```suggestion
You can always restore a Pytree out of your checkpoints by setting `target=None`.
```

(Maybe it's best to double check the spelling on JAX's site. Sometimes it's "PyTree" :thinking: )

Reply:
JAX site mostly uses "pytree": https://jax.readthedocs.io/en/latest/search.html?q=PyTree# 
Making it consistent here.

Reply:
Excellent, thanks!